examples/all-clusters-app/telink/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 #
-#    Copyright (c) 2022 Project CHIP Authors
+#    Copyright (c) 2022-2023 Project CHIP Authors
```

Comment:
These changes are not necessary: files are completely unchanged. This just creates change logs.

Reply:
Fixed